doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,30 @@
+# QoS policies
+
+Unlike other middlewares like for example [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html) iceoryx tries to keep things simple and offers only basic Quality of Service (QoS) settings. The QoS settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings, which are also available in DDS, are:
+
+* `PublisherOptions::historyCapacity`: This will enable late-joining subscribers to read the n last samples
```

Comment:
I would use the present tense here. "This enables late-joining subscribers to request the last n samples" Or most recent n samples. n of course being the historyCapacity, but it does probably not require mentioning.